### DISCUSSION TITLE
[Show] - Action buttons dropdown

### DISCUSSION THREAD

**Discussion by susanu** [Laravel-Backpack/community-forum#1118] [General] [answered]
Hi guys,

Is there any way we can have action buttons as a dropdown in a different position?
Sorry in advance if i bother you with these weird requests.
Something like this:
![Screenshot from 2024-08-13 19-11-20](https://github.com/user-attachments/assets/61a6a6c0-8331-4f59-b419-77e457356135)

**Comment by jcastroa87** [comment]
Hello @susanu 

There is not a function to do it easily, but you can publish and customize operation views:

```
php artisan backpack:publish crud/create
php artisan backpack:publish crud/edit
```

You will find the published views in /resources/views/vendor/backpack/crud/edit.blade.php

Cheers.

**Comment by susanu** [comment]
I don't want to do that, overrides for me are a no no.

**Comment by tabacitu** [comment, marked as answer]
There's no default way to do it @susanu , no. But if you really _really_ want to do that... you can [add a script widget to that page](https://backpackforlaravel.com/docs/6.x/base-widgets#script-1)... that has a bit of JS and moves them wherever you want 😉
